### ISSUE TITLE
bam file to the snap format?

### ISSUE THREAD

**Issue by shawnzhangyx** [r3fang/SnapTools#1]
Hi Rongxin, 

Thanks for making this great tool! With Snaptools, I wondering if there is option to start with the bam files, rather than fastq? For example, if I already performed the alignment, can I feed the bam files to the program and get the snap files?

**Comment by r3fang** [comment]
Yes, just skip the first two steps and start with ‘snaptools pre’ with unfiltered bam or bed as input. Bam or bed file needs to be sorted by read name before running this step.
